lib/openstax/salesforce/remote/book.rb

```diff
@@ -1,6 +1,7 @@
 module OpenStax::Salesforce::Remote
   class Book < ActiveForce::SObject
-    field :name,                      from: "Name"
+    field :name,                 from: "Name"
+    field :annualization_number, from: "Annualization_number__c", as: :decimal
```

Comment:
weird, what is "annualization" ?

Reply:
Apparently it's the way they estimate Fall/Spring students given the other term.